packages/bpk-component-tooltip/src/BpkTooltip.js

```diff
@@ -18,6 +18,12 @@
 
 /* @flow strict */
 
+/*
+This is the component for the tooltip that is displayed to users.
+
+The actual component that developers create (i.e. the default export from this package) is BpkTooltipPortal.
+*/
+
```

Comment:
I was confused by our weird naming – the component consumers interface with is actually `BpkTooltipPortal`, not `BpkTooltip`, so I added this as a hint for future people.

Another option would be to rename them:

`BpkTooltip` ➡️ `BpkTooltipInner`
`BpkTooltipPortal` ➡️ `BpkTooltip`

But I wanted to see what people think in the PR first. I don't think renaming them would be breaking as it's an internal implementation detail and would only break usage for people who were using the non-public interface.